### PR TITLE
Limit orders/design update 1

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

_This PR is part of a waterfall series to implement a design layout update._

- Replaces the background for dark/light mode

Colors I will gradually replace as variables in upcoming PRs.

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @fairlighteth , nice!

I will list issues that are not related to the current PR, but, it would be nice to pay attention to them in the next PRs: 
1. No cows in the dark mode (I believe, they will be added soon)
2. Color of the menu/tooltip in the dark mode is blending with the background color, so it would be great to change it
tooltip
 menu in the dark mode
3. Account sections color looks like almost blending with the bg-color in the light mode, so it would be also great to change sections' color in the next PRs 
account light mode
4. Boarder on the content pages like About/FAQ/T&C  is not visible in the dark mode, so it makes no sense to keep it
image
Thanks!

File: src/custom/theme/baseTheme.tsx [hunk 1]
```diff
@@ -109,16 +113,15 @@ export function colors(darkMode: boolean): Colors {
...
     body: {
       background: css`
-        background: url(data:image/svg+xml;base64,${background}) no-repeat 100% / cover fixed,
-          ${darkMode
-            ? 'linear-gradient(180deg,rgba(20, 45, 78, 1) 10%, rgba(22, 58, 100, 1) 30%)'
-            : 'linear-gradient(180deg,rgba(164, 211, 227, 1) 5%, rgba(255, 255, 255, 1) 40%)'};
+        ${darkMode
+          ? `
+          background-color: ${colorsTheme.blueDark1};
+          background-image: radial-gradient(50% 500px at 50% -13%,hsl(216deg 100% 20% / 70%) 0%,#071832 50%, #06162d 100%),radial-gradient(circle at -70% 50%,hsla(215,100%,20%,0.7) 0,transparent 50%);`
```
Comment (alfetopito) [inline src/custom/theme/baseTheme.tsx:123]: Nitpick: add space after commas to improve readability and reduce chance of error

**Review by alfetopito — APPROVED** [review]
Would only ask to make sure the final PR passes CI, as there are tests failing in this current iteration

Screen Shot 2022-10-18 at 16 10 01

Probably just a matter of outdated base branch

**Review by shoom3301 — APPROVED** [review]

File: src/custom/theme/baseTheme.tsx (hunk 1 above)
Comment (fairlighteth) [reply src/custom/theme/baseTheme.tsx]: Updated 👍🏼

**Pull request closed without merging**